src/core/CoolBoardSensors.cpp

```diff
@@ -223,7 +229,7 @@ float CoolBoardSensors::readSoilMoisture() {
 }
 
 float CoolBoardSensors::readWallMoisture() {
-  float val = 0;
+  int val = 0;
```

Comment:
why are you changing this ? It's wall moisture, not soil moisture. This shouldn't be here and probably introduce a bug.